hydra/test_utils/launcher_common_tests.py

```diff
@@ -399,6 +419,8 @@ def test_custom_task_name(
             prints="HydraConfig.get().job.name",
             expected_outputs=expected_name,
             filename=filename,
+            temp_remote_dir=self.temp_remote_dir(),
+            download=self.download(),
```

Comment:
the integration test function is very low level. I don't like that it needs to do the download and know about those things.
This diff now leaks details from the Ray launcher all the way into the core of the common testing in Hydra.